COMMON_ISSUES.md

```diff
@@ -44,6 +44,14 @@ to ensure playback:
   video.muted = true;
   ```
 
+Chrome 76+ Participants downgrade outgoing video bitrate for high audio bitrate limits
+--------------------------------------------------------------------------------------
+
+Because of this [bug](https://bugs.chromium.org/p/chromium/issues/detail?id=1002875), if you
+try to set the maximum outgoing audio bitrate (`maxAudioBitrate`) for the OPUS codec to values
+greater than or equal 64000 bps, then the outgoing video bitrate gets stuck at a very low value
+resulting in degraded quality for Participants subscribing to your VideoTrack.
+
```

Comment:
Because of this chrome [bug](https://bugs.chromium.org/p/chromium/issues/detail?id=1002875), if you
~try to~ set very high values for maximum outgoing audio bitrate (`maxAudioBitrate`) ~for the OPUS codec~ to values greater than or equal 64000 bps, then the outgoing video bitrate gets stuck at a very low value
resulting in degraded quality for Participants subscribing to your VideoTrack.

Reply:
couple of questions.

I don't have much context in this but on reading the referenced issue:

1) I did not find mention of OPUS. is this limited to OPUS codec?
2) I did not find any reference to 64K bps that you mention - is the issue limited to values greater than 64K?

Reply:
@makarandp0 I have linked to the paper doc in the PR description where I've noted my observations.

Reply:
ah, I missed the doc link. Thanks.

Can you also  change  'if you try to set very high values' => 
 - `if you set very high values`